LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/transition-in-hidden-page-expected.txt

```diff
@@ -0,0 +1,8 @@
+CONSOLE MESSAGE: Unhandled Promise Rejection: InvalidStateError: View transition was skipped because document visibility state is hidden.
```

Comment:
Shouldn't we be concerned about this?

Reply:
The test is expecting it `assert_equals(readyResult.status, "rejected");`

Reply:
Seems like the test is badly written and needs fixing if we get `harness Error (FAIL)` with the expected behavior.